src/mame/audio/nsub.cpp

```diff
@@ -0,0 +1,116 @@
+// license:BSD-3-Clause
+// copyright-holders:Zsolt Vasvari, MASH
+/*
+ *  N-Sub (upright) sound routines
+ */
+
+#include "emu.h"
+#include "includes/vicdual.h"
+
+/* output port 0x02 definitions - sound effect drive outputs */
+#define OUT_PORT_2_ALERT        0x01
+#define OUT_PORT_2_SONAR        0x02
+#define OUT_PORT_2_FIRE         0x04
+#define OUT_PORT_2_EXPLOSION    0x08
+#define OUT_PORT_2_EXPLOSION2   0x10
+#define OUT_PORT_2_BONUS        0x20
+#define OUT_PORT_2_SIGNAL       0x40
+#define OUT_PORT_2_SIREN        0x80
+
+
+#define PLAY(samp,id,loop)      samp->start( id, id, loop )
+#define STOP(samp,id)           samp->stop( id )
```

Comment:
These macros really don't add any value - they just obfuscate the code.  Also, macros are a recipe for confusing errors.  If you really want a helper like this, use an inline function.